dist/images/ovnkube.sh

```diff
@@ -3,28 +3,28 @@
 #set -euo pipefail
 
 # This script is the entrypoint to the image.
-# Supports version 3 daemonsets
+# Supports version 3, and 4 daemonsets
 #    $1 is the daemon to start.
-#        In version 3 each process has a separate container. Some daemons start
+#        In versions 3 and 4 each process has a separate container. Some daemons start
```

Comment:
"Each process has a separate container". You don't need to specify "In versions 3 and 4" since there aren't any other versions any more